scripts/e2e-android.sh

```diff
@@ -29,7 +29,7 @@ if [[ $FLOW == *"transaction"* || $FLOW == "e2e" ]]; then
 
   # Kill any existing Anvil process
   echo "Cleaning up any existing Anvil processes..."
-  ANVIL_PID=$(lsof -t -i:8545 -c anvil 2>/dev/null)
+  ANVIL_PID=$(lsof -t -i:8545 -a -c anvil 2>/dev/null)
```

Comment:
on local this kills my android sim. this fixes it.

Reply:
Nice, I was annoyed by this :)